src/PointwiseFunctions/AnalyticSolutions/Errors.hpp

```diff
@@ -0,0 +1,56 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Variables.hpp"
+#include "DataStructures/VariablesTag.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace Tags {
+/*!
+ * \brief Prefix indicating the difference between two values, such as betwen a
+ * numeric quantity and the corresponding quantity computed from an analytic
+ * solution.
+ */
+template <typename Tag>
+struct Difference : db::PrefixTag, db::SimpleTag {
```

Comment:
Call it Error not Difference; what other difference would there be?  I would have no idea of what, for example,`Difference<Lapse>` would mean...

Reply:
I was struggling for a good name...I like this suggestion!